### DISCUSSION TITLE
extended tweets for apiv2

### DISCUSSION THREAD

**Discussion by pradheepram** [tweepy/tweepy#2112] [Questions] [answered]
get_tweets is the same as streamingClient's filter, there is no way to get full tweet.

is the only way to go about this is to get tweet id from streamingclient because apiv1 does not support streaming now,

then use api1 get_status to get full tweet with tweet id because apiv2 does not support full tweets?

**Comment by Harmon758** [comment, marked as answer]
Twitter API v2 natively supports Tweets with up to 280 characters and has no extended Tweet concept.
